study-builder/studies/pancan/patches/unique-cancer-validation.conf

```diff
@@ -0,0 +1,22 @@
+{
+  "studyGuid": "cmi-pancan",
+  "questionStableId": "PRIMARY_CANCER_SELF",
+  "uniqueRule":
+    {
+      "ruleType": "UNIQUE",
+      "allowSave": false,
```

Comment:
allowSave is a non-nullable attribute in table: validation with a default value: false
Want to clarify that if the value is NOT set for any validation, its defaulting to "false" .
If you're expecting front-end to check this value and behave differently I don't think the approach will work.
If front-end behavior is same as every other validation which has default value of "false", we can remove explicitly setting it to "false" here.

Reply:
Set allowSave=true (as for all other rules).